### ISSUE TITLE
Apply HuggingFace accelerate for multi-GPU training on 2x H200

### ISSUE THREAD

**Issue by coport-uni** [coport-uni/FR5ControllerVLA#30]
## Summary
Integrate HuggingFace `accelerate` into the LeRobot training code so that
FR5 policy training can utilize multiple GPUs (specifically 2x H200),
following the official LeRobot multi-GPU training guide:
https://huggingface.co/docs/lerobot/multi_gpu_training

## Motivation
Current training runs on a single GPU. We have 2x H200 available and want
to cut wall-clock training time for FR5 VLA policies.

## Scope
- Apply `accelerate` to the training entry point
  (`lerobot-train` / `src/lerobot/scripts/lerobot_train.py`) and the
  training loop: `Accelerator` init, `accelerator.prepare(...)` for
  model/optimizer/dataloader, main-process-only checkpointing/logging,
  cross-process metric reduction.
- Provide/document an `accelerate` config for 2x H200
  (`num_processes=2`, bf16 or fp16, multi_gpu).
- Update the FR5 train launch script(s) to use
  `accelerate launch --num_processes=2 ...`.

## Acceptance Criteria
- [ ] `accelerate` is applied to the training code per the LeRobot guide.
- [ ] Training launches successfully with
      `accelerate launch --num_processes=2` on 2x H200.
- [ ] Training runs for **~5 minutes with no errors**.
- [ ] `nvidia-smi` confirms **both** H200 GPUs are actively utilized
      (non-zero util% and memory) during the run; evidence captured
      (log or screenshot).
- [ ] Ruff check/format pass on all modified files.

## Reference
- https://huggingface.co/docs/lerobot/multi_gpu_training

**Comment by coport-uni** [comment]
Done in 048b3ad5.

`accelerate launch --multi_gpu --num_processes=2 --mixed_precision=bf16 $(which lerobot-train)` drives 2x H200 end-to-end via the existing `7__train_act.sh`. `lerobot_train.py` got a small `InitProcessGroupKwargs(timeout=1h)` patch so slow one-time startup stalls do not trip the 10-minute default NCCL watchdog.

**In-container verification:** ACT on FR5 (`coport-uni/FR5_pick_red_colored_marker_to_box`) reached step 460 in ~6 min, loss 24.25 → 1.69, no errors. Both GPUs held ~44 GB and alternated 95-100% util via `nvidia-smi`. Effective batch size logged as `64 x 2 = 128`.

**Container-only note** (not in the production launch script): NCCL P2P/CUMEM transport hangs silently on the first allreduce inside this Docker container; exporting `NCCL_P2P_DISABLE=1 NCCL_SHM_DISABLE=1` forces the socket fallback. Host NUC is unaffected so `7__train_act.sh` keeps the defaults.

Evidence under `claude_test/accelerate_mgpu_evidence/` (gitignored `*.log` files): `train.log`, `gpu_samples.log`, `smoke_nccl_socket_ok.log`.
